test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -221,7 +221,7 @@ export default class BaseFormObject {
   }
 
   hasErrorBox() {
-    return within(this.el).getByRole('alert').exists;
+    return Selector('[role="alert"]').exists;
```

Comment:
Is this change needed for the tests? We should prefer the `getByRole` selector whenever possible

Reply:
`within(this.el).getByRole('alert').exists;` never works when the alert doesn't exist. The `getByRole` requires the element to be present to work. So in case it doesn't exist, it throws an error.
So far, this was only used to check if the alert exists. But in this PR, I had to test for alert doesn't exist and this doesn't work.

Reply:
Let's change it to `queryByRole`, should work then